fetch.bs

```diff
@@ -2049,6 +2049,36 @@ clearly stipulates that <a>connections</a> are keyed on
 <!-- See https://github.com/whatwg/fetch/issues/114#issuecomment-143500095 for when we make
      WebSocket saner -->
 
+<h3 id=http-cache-partitions>HTTP cache partitions</h3>
+
+<p>To <dfn export>determine the HTTP cache partition</dfn>, given <var>request</var>, run these
+steps:
+
+<ol>
+ <li>
+  <p>Let <var>environmentSettings</var> be <var>request</var>'s <a for=request>client</a>.</p></li>
+
+ <li>
+  <p>If <var>environmentSettings</var> is null, then return null.</p></li>
+
+ <li>
+  <p>Let <var>topLevelOrigin</var> be <var>environmentSettings</var>'s top-level origin and
+  <var>origin</var> be its <a for="environment settings object">origin</a>.</p></li>
+
+ <li>
+  <p>Let <var>topLevelOriginAgentClusterKey</var> be the result of
+  <a href="https://html.spec.whatwg.org/#obtain-agent-cluster-key">obtaining an agent cluster key</a>,
```

Comment:
These should also be updated to `<a>obtaining an agent cluster key</a>`, but will need HTML-side updates before that correctly auto-links. No action for now but something to keep in mind before the editors do the full merge.

Reply:
I created https://github.com/whatwg/html/pull/5354 to discuss this.